using_images/other_images/jenkins.adoc

```diff
@@ -270,6 +270,10 @@ that extends this image and updates plugins in the custom image after
 the initial startup, by default they are not copied over, unless you set this
 environment variable to `true`.
 
+* `ENABLE_FATAL_ERROR_LOG_FILE` (default: `false`) 
++
+When running this image with an OpenShift persistent volume claim for the Jenkins config directory, this environment variable allows the fatal error log file to persist if a fatal error occurs. The fatal error file will be located at `/var/lib/jenkins/logs`.
```

Comment:
s/OpenShift//

Reply:
@bparees sorry, what are you wanting changed?

Reply:
delete the word openshift

Reply:
(and change the "an" to an "a")